src/app/pagamentoplanos/pagamentoplanos.module.ts

```diff
@@ -8,6 +8,9 @@ import { HeaderlogModule } from '../headerlog/headerlog.module';
 import { HeaderUserComumModule } from '../headerusercomum/headerusercomum.module';
 import { FooterModule } from '../footer/footer.module';
 import { PagamentoplanosComponent } from './component/pagamentoplanos.component';
+import { ExternalService } from './service/external.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8398929)
```suggestion
import { ExternalService } from "./service/external.service";
```